lib/rubocop/cop/lint/multiple_comparison.rb

```diff
@@ -7,25 +7,26 @@ module Lint
       # multiple value. However, we can't use the comparison in Ruby. However,
       # the comparison is not syntax error. This cop checks the bad usage of
       # comparison operators.
+      # It allows multiple comparison using `&`, `|`, and `^` set operation operators.
```

Comment:
I'm a bit confused why this is a problem at all - looking at the defintion of the cop it shouldn't be looking at something like this at all. (unless I'm missing the point here)